x/concentrated-liquidity/pool.go

```diff
@@ -69,15 +69,16 @@ func (p Pool) GetTotalShares() sdk.Int {
 	return sdk.Int{}
 }
 
-func (k Keeper) SwapOutAmtGivenIn(ctx sdk.Context, tokenIn sdk.Coin, tokenOutDenom string, swapFee sdk.Dec, minPrice, maxPrice sdk.Dec, poolId uint64) (tokenOut sdk.Coin, err error) {
-	tokenInCoin, newLiquidity, newCurrentTick, newCurrentSqrtPrice, err := k.CalcOutAmtGivenIn(ctx, tokenIn, tokenOutDenom, swapFee, minPrice, maxPrice, poolId)
-	if err != nil {
-		return sdk.Coin{}, err
-	}
+// TODO: implement this
```

Comment:
?

Reply:
adam's taking this, he'd be creating an issue soon to track this!